doc/modules/preprocessing.rst

```diff
@@ -34,8 +34,8 @@ standard deviation.
 
 For instance, many elements used in the objective function of
 a learning algorithm (such as the RBF kernel of Support Vector
-Machines or the l1 and l2 regularizers of linear models) assume that
-all features are centered around zero and have variance in the same
+Machines or the l1 and l2 regularizers of linear models) may assume that
+all features are centered around zero and/or have variance in the same
```

Comment:
Now a days, I prefer to choose one of the conjunctions:

```suggestion
all features are centered around zero or have variance in the same
```

For reference, here is a [wikipedia page](https://en.wikipedia.org/wiki/And/or) discussing And/or.